File: problem_5/python/problem5.py
#!/usr/bin/python

def div_11_to_20(divided):
  return all([not bool(divided % x) for x in xrange(11,20+1)])

if __name__ == "__main__":
  count = 2520
  while div_11_to_20(count) == False:
    count += 2520

  print "%s" % count

File: problem_5/rust/problem5.rs

fn div_11_to_20(divided:usize) -> bool {
  (11..21).all(|i| divided % i == 0)
}

fn main() {
    let mut count:usize = 2520;
    while div_11_to_20(count) == false {
        count += 2520;
    }

    println!("{}", count);
}
